modules/gitops-configuring-argo-cd-using-dex-github-conector.adoc

```diff
@@ -0,0 +1,33 @@
+[id="configuring-argo-cd-using-dex-github-connector_{context}"]
+= Configuring Argo CD SSO using Dex GitHub connector
+
+[discrete]
+.Procedure
+
+. Register the application in the identity provider as explained link:https://argo-cd.readthedocs.io/en/stable/operator-manual/user-management/#1-register-the-application-in-the-identity-provider[here]
+. Update the Argo CD CR.
```

Comment:
What's a `CR`? If this acronym hasn't been defined yet, include the full name here followed by `(CR)` if you're going to reference the acronym later.

Reply:
@bburt-rh This page is not included in the final output. so not making the changes now.